docker-compose.yml

```diff
@@ -95,15 +95,25 @@ services:
       - mysql
     depends_on:
       - mysql
+      - wikibase
```

Comment:
what is docker-compose.yml used for ? 
if compose-dev is for local purposes.

Reply:
see README section "Start up the containers"